tools/vscode/ddlog-language/package.json

```diff
@@ -42,7 +43,6 @@
     "devDependencies": {
         "gulp": "^4.0.2",
         "gulp-json5-to-json": "^0.2.1",
-        "gulp-merge-json": "^1.3.1",
-        "vsce": "^1.84.0"
+        "gulp-merge-json": "^1.3.1"
```

Comment:
I thought vsce is actually required. The installation instructions use vsce.
Please note that I modified the installation to use locally-installed versions of these tools instead of the global ones as in the initial implementation (i.e., no -g flag supplied to npm when installing).